data/base-hunting.yaml

```diff
@@ -399,6 +399,12 @@ hunting_zones:
   - 1446
   - 1447
   - 1448
+  fell_hogs:
```

Comment:
Please also include a link to elanthipedia and approximate hunting range (see line 408 for an example)